frontend/src/redux/reducers/appReducer.ts

```diff
@@ -5,7 +5,7 @@ const initialState: AppState = {
   notifications: [],
   forceComponentsUpdate: 0,
 };
-
+//eslint-disable-next-line @typescript-eslint/default-param-last
```

Comment:
disabled this because was causing error in frontend/src/redux/store/store.ts  , the createStore function